README.md

```diff
@@ -31,14 +31,7 @@ Hello World! ヾ(☆◡☆) This is a mentorship-based remote learning platform
 npm install
 ```
 
-### 2. Setting Up Firebase Authentication
-
-### 3. Setting Up Firebase Firestore Database
-#### API Routes & Endpoints
-All API routes are in `pages/api/`.
-
-
-### 4. Running the Project
+### 2. Running the Project
```

Comment:
We switched the order of these numbers. When you get a chance pull the latest main with the updated numbers and resolve merge conflicts